### PR TITLE
Shorten challenge admin docs

### PR DESCRIPTION
**Pull request by thomasyu888**
This is an effort to remove language from the docs that are specific to our internal use.

### CODE REVIEW

**Comment by thomasyu888** [comment]
Also, its not completed. Just opened PR to remind myself of this task

File: articles/challenge_administration.md [hunk 1]
```diff
@@ -6,84 +6,67 @@ category: howto
...
-This [createChallenge.py](https://github.com/Sage-Bionetworks/DREAM-Utilities/blob/master/createChallenge.py) script sets up the challenge infrastructure (steps A1-5).  
 
-**For all DREAM challenges, please contact the Synapse Access and Compliance Team (act@sagebase.org) for step A4!**
+## A - Challenge Space Creation
 
-### 1 - Creating Teams
+Steps A1-5 below will be completed for you by using the `createchallenge` command after installing the [challengeutils](https://github.com/Sage-Bionetworks/challengeutils) python package.  This documentation below serves to provide detailed descriptions for each of the steps.
 
-In Synapse, Teams are publicly viewable groups of users.  Teams may be given access to projects, data files, and Evaluation submission queues as a group.  Using a Team for a challenge allows you to see who is participating and to conveniently control access to challenge resources.  **All challenges require a challenge participant, challenge pre-registrant, and administrator team.**
+**For all DREAM challenges, please contact the Synapse Access and Compliance Team (act@sagebase.org) for step A4!**
```
Comment (kelshmo) [inline articles/challenge_administration.md]: what is step A4 - can we explicitly state it here?

**Review by kelshmo — COMMENTED** [review]
In-progress review - I want to pull into the dev build to look at format

**Review by tschaffter — CHANGES_REQUESTED** [review]
Review (part 1/2)

**Review by tschaffter — CHANGES_REQUESTED** [review]

**Review by kelshmo — APPROVED** [review]

**Pull request merged**